docs/openapi/components/schemas/common/AgricultureProduct.yml

```
@@ -108,27 +108,19 @@ properties:
 additionalProperties: true
 example: |-
   {
-    "type": "AgricultureProduct",
+    "type": ["AgricultureProduct"],
     "upc": "033383401508",
     "plu": "94225",
     "gtin": "033383401508",
     "product": {
-      "type": "Product",
+      "type": ["Product"],
       "manufacturer": {
-        "type": "Person",
-        "firstName": "Itzel",
-        "lastName": "Leannon",
-        "email": "Camila_Braun51@jimbofarm.org",
-        "phoneNumber": "555-900-1599",
-        "worksFor": {
-          "type": "Organization",
-          "name": "Jimbo's Awesome Farm",
-          "description": "Organic fruits & vegetables",
-          "email": "Chloe.Ondricka42@jimbofarm.org",
-          "phoneNumber": "555-496-7149",
-          "faxNumber": "555-209-6489"
-        },
-        "jobTitle": "District Quality Associate"
+        "type": ["Organization"],
+        "name": "Jimbo's Awesome Farm",
+        "description": "Organic fruits & vegetables",
+        "email": "Chloe.Ondricka42@jimbofarm.org",
```

I'll grant you, `jimbofarm.org` doesn't exist today, but that doesn't guarantee it never will, nor that this sample data won't lead to unknowable problems if that domain is ever live. Still, please, let's always use `.example` or `.example.org` or the like for TLDs in sample data. `jimbofarm.example` or ``jimbofarm.example.org` are both viable changes.
```suggestion
        "email": "Chloe.Ondricka42@jimbofarm.example.org",
```